### ISSUE TITLE
Deprecated project still part of django-cms documentationa dn examples

### ISSUE THREAD

**Issue by Bartvaderkin** [divio/aldryn-apphooks-config#76]
https://docs.django-cms.org/en/latest/how_to/namespaced_apphooks.html

**Comment by yakky** [comment]
@Bartvaderkin yes, I am going to keep this up to date because it's required by djangocms-blog (and the only implementation of the django CMS apphook config)
